models/articles.models.js

```diff
@@ -1,18 +1,38 @@
 const db = require("../db/connection")
 
 function selectArticleById(article_id) {
-  return db
-    .query(`SELECT * FROM articles WHERE article_id = $1`, [article_id])
-    .then(({ rows }) => {
-      if (rows.length === 0) {
-        return Promise.reject({
-          status: 404,
-          message: "article does not exist",
-        })
-      } else {
-        return rows[0]
-      }
-    })
+  return (
+    db
+      .query(
```

Comment:
If you want to shorten this query back down, you can put `SELECT articles.*, ` and then your comment_count